packages/davinci-client/src/lib/collector.types.ts

```diff
@@ -34,14 +35,15 @@ export type ActionCollectorTypes =
   | 'ActionCollector';
 export interface ActionCollector {
   category: 'ActionCollector';
+  error: string | null;
   type: ActionCollectorTypes;
   id: string;
   name: string;
   output: {
     key: string;
     label: string;
     type: string;
-    url?: string;
+    url?: string | null;
```

Comment:
This is a bit of a redundant type?

Reply:
Anytime I'm explicitly using setting a value to be "no value", I'm using `null`. This helps differentiate between `undefined`, which a interpret as a value was expected, but not value was provided, and `null` which means we intentionally provided a "no value" value. I hope that makes sense. But, I think you have some changes that may render this particular change moot, yeah?

Reply:
Yeah, I may have changed this. The redundant part is the `?` in the url.